xdsl/interactive/app.py

```diff
@@ -100,9 +107,10 @@ class InputApp(App[None]):
     Reactive variable used to save the current state of the modified Input TextArea
     (i.e. is the Output TextArea).
     """
-    pass_pipeline = reactive(tuple[type[ModulePass], ...])
-    """Reactive variable that saves the list of selected passes."""
+    pass_pipeline = reactive(tuple[ModulePass, ...])
+    # pass_pipeline = reactive(tuple[dict[type[ModulePass], PipelinePassSpec], ...])
```

Comment:
as discussed over zulip, it feels like this is the way to go, since ModulePass is not frozen, and we can't guarantee that they won't be in a weird state after being run once

Reply:
Done.